test/replication-luatest/gh_7515_sync_node_sees_leader_hang_test.lua

```diff
@@ -58,7 +58,7 @@ g.test_notice_leader_hang_during_sync = function(cg)
     -- restart. Generate transactions, to make sure heartbeats do not interfere
     -- with transactions, and make relay sleep after each sent row to make sure
     -- it takes server2 long enough to sync.
-    local old_term = cg.server1:exec(function()
+    local old_term, tweaks_default = cg.server1:exec(function()
```

Comment:
Is the test changed for EE? That doesn't look good that EE CI depends on these hacks in CE tests. I suggest to find a way not to make CE tests so fragile. Is it possible for this "feature"?

Reply:
Now that's everything is implemented in CE, the test is changed for CE.

I have to tweak batch size, because the test requires sending multiple batches with timeouts in between (so that server sync takes some time), and I wanted to send as little data as possible.